packages/app-apw/src/hooks/useCmsModels.ts

```
@@ -0,0 +1,36 @@
+import { LIST_CMS_MODELS, ListCmsModelsQueryResponse } from "~/graphql/workflow.gql";
+import { CmsModel } from "~/types";
+import { useI18N } from "@webiny/app-i18n/hooks/useI18N";
+import { useState } from "react";
+import { useQueryLocale } from "@webiny/app-headless-cms/admin/hooks";
+import get from "lodash/get";
+
+interface UseCmsModelsResult {
+    models: CmsModel[];
+    loading: boolean;
+}
+
+export const useCmsModels = (): UseCmsModelsResult => {
+    const { getCurrentLocale } = useI18N();
+    const locale = getCurrentLocale("content") || "en-US";
```

I wouldn't use any fallback here (`en-US`). Seeing that `en-US` is _not_ always the default locale for our clients, this can cause hard to debug issues. It's better to throw an error if the locale is not available.